internal/storage/persistence/ent/client/client.go

```diff
@@ -185,7 +185,10 @@ func (c *Client) UpdateAIPLocationID(ctx context.Context, aipID, locationID uuid
 	return nil
 }
 
-func (c *Client) ListWorkflows(ctx context.Context, f *persistence.WorkflowFilter) (goastorage.AIPWorkflowCollection, error) {
+func (c *Client) ListWorkflows(
+	ctx context.Context,
+	f *persistence.WorkflowFilter,
+) (goastorage.AIPWorkflowCollection, error) {
```

Comment:
Looks like golangci-lint is not properly failing the CI workflow again after the latest changes.

Reply:
Fixed in 95c9b62 :)